test/unit/views/v2/LiabilityViewSpec.scala

```diff
@@ -202,9 +226,126 @@ class LiabilityViewSpec extends ViewSpec {
         Cases.activityTabViewModel,
         activityForm,
         None,
-        uploadAttachmentForm
+        uploadAttachmentForm,
+        emptyKeywordsTabViewModel,
+        keywordForm
       ))
       doc should containElementWithID("sample_status_tab")
     }
+
+    "render Keywords tab when showRulingAndKeywords is true" in {
+      val c = aLiabilityCase(withReference("reference"), withLiabilityApplication()).copy(status = CaseStatus.OPEN)
+      val doc = view(liabilityView(
+        LiabilityViewModel.fromCase(c, Cases.operatorWithAddAttachment),
+        None,
+        None,
+        sampleStatusTabViewModel,
+        Cases.activityTabViewModel, activityForm,
+        Cases.attachmentsTabViewModel.map(_.copy(attachments = Seq(Cases.storedAttachment), letter = Some(Cases.letterOfAuthority))),
+        UploadAttachmentForm.form,
+        emptyKeywordsTabViewModel,
+        keywordForm
+      ))
+      doc should containElementWithID("keywords_tab")
+    }
+
+    "not render Keywords tab when showRulingAndKeywords is false" in {
+      val c = aLiabilityCase(withReference("reference"), withLiabilityApplication()).copy(status = CaseStatus.NEW)
+      val doc = view(liabilityView(
+        LiabilityViewModel.fromCase(c, Cases.operatorWithoutPermissions),
+        None,
+        None,
+        sampleStatusTabViewModel,
+        Cases.activityTabViewModel,
+        activityForm,
+        None,
+        uploadAttachmentForm,
+        emptyKeywordsTabViewModel,
+        keywordForm)
+      )
+      doc shouldNot containElementWithID("keywords_tab")
+    }
+
+    "render the action this case button for new case" in {
```

Comment:
all test for buttons was moved to new partial view and with separate test cases here https://github.com/hmrc/tariff-classification-frontend/blob/master/test/unit/views/partials/liabilities/CaseActionButtonsViewSpec.scala make sure you cleanup here as well

Reply:
removed button tests from liabilityviewspec